java/kotlin-extractor/src/main/kotlin/KotlinUsesExtractor.kt

```diff
@@ -414,31 +414,35 @@ open class KotlinUsesExtractor(
 
     // `typeArgs` can be null to describe a raw generic type.
     // For non-generic types it will be zero-length list.
-    fun addClassLabel(cBeforeReplacement: IrClass, argsIncludingOuterClasses: List<IrTypeArgument>?, inReceiverContext: Boolean = false): TypeResult<DbClassorinterface> {
+    private fun addClassLabel(cBeforeReplacement: IrClass, argsIncludingOuterClasses: List<IrTypeArgument>?, inReceiverContext: Boolean = false): TypeResult<DbClassorinterface> {
```

Comment:
Recommend renaming the direct parameter `argsIncludingOuterClasses` to `argsBeforeReplacement`, pulling out `tryReplaceAndroidSyntheticClass` and `tryReplaceParcelizeRawType` into a function, and generally avoiding having `c` and `argsIncludingOuterClasses` sitting around in scope but inappropriate to use -- it's better if the 'wrong' symbols have names like `BeforeReplacement` to avoid someone accidentally using them instead of the replaced version.